### PR TITLE
add GetTimeZonesForRegion

### PR DESCRIPTION
**Pull request by fabriziomoscon**
First attempt...

Retrieve the timezone of a string phone number passed. It uses the same data used by googlei18n/libphonenumber

related to this discussion #30

---

This change is [Reviewable](https://reviewable.io/reviews/ttacon/libphonenumber/31)

### CODE REVIEW

**Comment by fabriziomoscon** [comment]
does this PR need more work @ttacon ?

File: countryCodeToTimeZones.go [hunk 1]
```diff
@@ -0,0 +1,2261 @@
+package libphonenumber
+
+import "strconv"
+
+// TODO(ttacon): do we need to worry about load factor?
```
Comment (ttacon) [inline countryCodeToTimeZones.go]: I would been somewhat concerned. Could you test some build times with and w/o this file?

**Comment by ttacon** [comment]
Hey! Sorry, just a quick question about load time!

**Comment by fabriziomoscon** [comment]
hi,
I have just copied it from this:
https://github.com/ttacon/libphonenumber/blob/5cb77679a4c77d45f2496c9ed8e60b5eec03bb47/countrycodetoregionmap.go#L3
I was not quite sure what the comment meant. Maybe you could remember why did you add the comment in the first place.

**Comment by fabriziomoscon** [comment]
@ttacon the build time for this branch is not affected by the new file, so I remove the comment. Please let me know if there anything else to do before merging

File: countryCodeToTimeZones.go [hunk 2]
```diff
@@ -0,0 +1,2259 @@
+package libphonenumber
+
+import "strconv"
+
+const MAX_REGION_CODE_LENGTH = 7
+
+var CountryCodeToTimeZones = map[int][]string{
```
Comment (ttacon) [inline countryCodeToTimeZones.go]: Can we add a comment here as to what the structure of this mapping means semantically?

**Comment by ttacon** [comment]
If you can add a comment to the semantic structure of the `CountyCodeToTimeZones` map, I'll merge this after!

**Comment by fabriziomoscon** [comment]
@ttacon I have added this comment:
```
// This structure maps telephone number digits to a particular timezone,
// because a timezones could represent many prefixes, there could be many
// entries for the same timezone.
// For example this is the case for America/New_York.
```
and squashed the 3 commits.

**Comment by ttacon** [comment]
Thanks for the contribution!

**Pull request merged**